JPS_Ontology/ontology/ontoassetmanagement/OntoAssetManagement.csv

```diff
@@ -30,8 +30,38 @@ UnitPrice,Class,Price,IS-A,,,,Price per individual product or service.,https://w
 DiscountedUnitPrice,Class,Price,IS-A,,,,Discounted price of one unit of a product or service.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Discounted unit price
 TotalDiscountedPretaxPrice,Class,Price,IS-A,,,,Total price of a purchase including discount and excluding tax.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Total discounted pretax price
 HomeTotalDiscountedAfterTaxPrice,Class,Price,IS-A,,,,"Total Price of a purchase including tax and discount, converted to a local currency.",https://www.theworldavatar.com/kg/ontoassetmanagement/,Home total discounted after-tax price
-ServiceCategory,Class,https://purl.org/p2p-o/item#ItemAttribute,IS-A,,,,An attribute of an item that describes the service category it falls under.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Service category
-ServiceCode,Class,https://purl.org/p2p-o/item#ItemAttribute,IS-A,,,,An attribute of an item that describes the service code it falls under.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Service code
+MaintenanceSchedule,Class,,,,,,"A maintenance schedule tells and records when an asset is scheduled for maintenance, when was it last maintained, who performs the maintenance etc.",https://www.theworldavatar.com/kg/ontoassetmanagement/,Maintenance schedule
+MaintenanceTask,Class,,,,,,"A small, identifiable piece of work that indicates the actions one must take to complete a maintenance work order.",https://www.theworldavatar.com/kg/ontoassetmanagement/,Maintenance task
+Event,Class,,,,,,"A thing that happens or takes place, especially one of importance.",https://www.theworldavatar.com/kg/ontoassetmanagement/,Event
+AssetUpdateHistory,Class,,,,,,A record of updates applied to the information of an asset.,https://www.theworldavatar.com/kg/ontoassetmanagement/,Asset update history
+Furniture,Class,,,,,,"Objects intended to support various human activities such as seating (e.g., stools, chairs, and sofas ), eating ( tables ), storing items, working, and sleeping (e.g., beds and hammocks ). ",https://www.theworldavatar.com/kg/ontoassetmanagement/,Furniture
```

Comment:
I guess you are missing the definition of Furniture as subclass of Element

Reply:
Oh yes. Sorry. Missed that out.